### DISCUSSION TITLE
Ebus Polarity for Vaillant Aquaflow exclusive controller?

### DISCUSSION THREAD

**Discussion by nkrepo** [john30/ebusd#679] [General]
I have a "Vaillant Aquaflow exclusive" unit that has ebus port. Unfortunately the ebus polarity is not marked on the connector.  I look into the manual but also the polarity is not reported. I tried to measure the voltage with a multi-meter but it reported zero volts at DC,  so I am clueless, any help would be appreciated.

I am attaching photos from the controller and the corresponding section in the manual.

![image](https://user-images.githubusercontent.com/8989693/196436145-18a9e6a6-15e2-4b89-bcef-3dd439c0ae1e.png)

![image](https://user-images.githubusercontent.com/8989693/196436621-4f0fe713-c14d-4310-a8a3-049bf27fcd9c.png)

![image](https://user-images.githubusercontent.com/8989693/196436658-2e271903-9ebe-43ec-9a74-0f65b4b7f2ad.png)

**Comment by cyberthom42** [comment]
You do not need to care about polarity, it does not matter. It is for example described in the manual for the controller (700 or 720), search for "polung"

**Reply by nkrepo** [reply]
Thank you for the information , i just started getting familiar with ebus.

**Comment by nkrepo** [comment]
@cyberthom42 can you point me at the manual you are looking ?
